src/Adapters/Phpunit/Printer.php

```diff
@@ -58,7 +59,7 @@ final class Printer implements \PHPUnit\TextUI\ResultPrinter
      *
      * @throws \ReflectionException
      */
-    public function __construct(\Symfony\Component\Console\Output\ConsoleOutputInterface $output = null, bool $verbose = false, string $colors = 'always')
+    public function __construct(ConsoleOutputInterface $output = null, bool $verbose = false, string $colors = 'always')
```

Comment:
Can't you intercept the output to add the behaviour you want?

Reply:
@nunomaduro that could be possibly but very difficult because the `Output` class is in a very low level and it's just receiving basic calls like:

`$this->output->write("  $diff");` or `$this->output->writeln('');`

So, I might end up having to check every call to `write` with a regex to determine if that's where I need to add the link. That's particularly challenging since people write tests in different formats:

`public function testSomething` or 

```
/** @test */
function does_something()
```

Also any changes to the way Collision formats the output might cause the regex to fail or match when it shouldn't, etc. 

If you'd like to have a quick look to my PR in Enlighten, I'm basically just overwriting the `writeError` method to call the parent and add the link below: https://github.com/StydeNet/enlighten/pull/25/files#diff-3a6dea9d73af1b5cf512525bb0b5d388e331b417941e7716576dc4e763cbfcaaR14

I reckon that's also prone to problems if Collision changes but even so it'd be much easier to adjust the code.

I tried to make as few changes as possible to Collision to support this feature.